Dockerfile

```diff
@@ -18,9 +18,19 @@ COPY pkg/ pkg/
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# Compose the final image
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5-218
```

Comment:
What about to use `registry.access.redhat.com/ubi8/ubi-minimal:8.6` or `registry.access.redhat.com/ubi8/ubi-minimal:8.5` ?

Reply:
We're now using the latest version.